deepchem/feat/molecule_featurizers/one_hot_featurizer.py

```diff
@@ -16,14 +17,14 @@
 ]
 
 
-class OneHotFeaturizer(MolecularFeaturizer):
-  """Encodes SMILES as a one-hot array.
+class OneHotFeaturizer(Featurizer):
+  """Encodes SMILES or any arbitrary string as a one-hot array.
 
-  This featurizer encodes SMILES string as a one-hot array.
+  This featurizer encodes either a SMILES string or any arbitrary string as a one-hot array.
```

Comment:
Could we expand on the description?  I don't really have any idea what it means.  Are we talking about one-hot encoding of all unique strings?  Or strings drawn from some vocabulary?  Or characters within strings?  The description "either a SMILES string or any arbitrary string" is also kind of confusing.  It implies SMILES strings are handled differently from other strings, but it doesn't say how.

Reply:
Characters within strings! 

Fixed the comment to add more detail. Hope that helps.

Reply:
Much clearer now.  Thanks!